mLRS/CommonRx/mavlink_interface_rx.h

```diff
@@ -183,4 +243,41 @@ if(txbuf>90) dbg.puts("-20 "); else dbg.puts("+-0 ");*/
 }
 
 
+void MavlinkBase::generate_rc_channels_override(void)
+{
+    fmav_msg_rc_channels_override_pack(
+        &msg_serial_out,
+        RADIO_STATUS_SYSTEM_ID, MAV_COMP_ID_TELEMETRY_RADIO, // sysid, SiK uses 51, 68
```

Comment:
The FC autopilot will probably only accept the override message if the system ID matches, so we should use 255 here which is what the ground control system is using by default. 68 is still good for the component ID.

Reply:
you are correct, good point
the correct value would be AP's MY_GCS parameter, but for the moment 255 should indeed do it
https://github.com/ArduPilot/ardupilot/blob/master/libraries/GCS_MAVLink/GCS_Common.cpp#L3461-L3463